README.md

```diff
@@ -1,84 +1,40 @@
+# Welcome Gophers!
 
+Welcome to the Athens project! We're building all things Go package repository in here. 
 
-# Athens
+1. [Package Registry](https://github.com/gomods/athens/wiki/The-Central-Package-Registry-(Olympus))
+2. [Edge Proxy](https://github.com/gomods/athens/wiki/Proxies-(Zeus))
 
-*This is a very early alpha release, and the API will be changing as the proxy API changes.*
-_Do not run this in production. This warning will be changed or removed as the project and the proxy API changes._
+If you want more of a tl;dr on the project, check out [this quick introduction](https://medium.com/@arschles/project-athens-c80606497ce1)
 
-Athens is a proxy server for [vgo modules](https://github.com/golang/go/wiki/vgo). It implements the download protocol specified [here](https://research.swtch.com/vgo-module) (under "Download Protocol"), and a few additional API endpoints to make it more useful. See [API.md](./API.md) for more information.
+# Project Status
 
-# Architecture
+Project Athens is in a very early alpha release and everything might change.
+Don't run it in production, but do play around with it and [contribute](#contributing)
+when you can!
 
-Athens is composed roughly of three logical pieces. The below list contains links to a description of each:
+# More Details Please!
 
-* [Module proxy](./PROXY.md)
-* [Module registry](./REGISTRY.md)
-* [CLI](./CLI.md)
+Although the project is in development, here's where we're going:
 
-# Development
-
-The server is written using [Buffalo](https://gobuffalo.io/), so it's fairly straightforward
-to get started on development. You'll need Buffalo v0.11.0 or later to do development on Athens.
-
-Download [v0.11.0](https://github.com/gobuffalo/buffalo/releases/tag/v0.11.0) or later, untar/unzip the binary into your PATH, and then run the
-following to run [Olympus](https://github.com/gomods/athens/wiki/The-Central-Package-Registry-(Olympus)):
-
-```console
-cd cmd/olympus
-buffalo dev
-```
-
-... and the following to run [Athens](https://github.com/gomods/athens/wiki/Proxies-(Athens):
-
-```console
-cd cmd/proxy
-buffalo dev
-```
-
-After you see something like `Starting application at 127.0.0.1:3000`, the server
-is up and running. As you edit and save code, Buffalo will automatically restart the server.
-
-## Dependencies and Set-up
+The package registry and the edge proxy both implement the [vgo download protocol](https://medium.com/@arschles/project-athens-the-download-protocol-2b346926a818), but each one
+is intended for different purposes.
 
-To run the development server, or run tests (tip: run `make test` to easily run tests), you'll need a running MongoDB server. We plan to add more service dependencies in the future, so we are using [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/) to create and destroy
-development environments.
+The registry will be hosted globally, and will be "always on" for folks. Anyone will be able to 
```

Comment:
I know this is merged, but can we stick with names Olympus/Zeus or at least state them besides using 'registry', 'proxy'... so new people are not confused by all this new vocabulary